kubesignin/templates/kubesignin.yaml

```diff
@@ -53,19 +53,9 @@ spec:
         - name: http
           containerPort: 80
 
-{{- if .Values.Dex.Tls }}
-        volumeMounts:
-        - name: tls
-          mountPath: /etc/dex/tls
-{{- end }}
-
         env:
         - name: DEX_ISSUER
           value: {{.Values.Dex.Issuer}}
-{{- if .Values.Dex.Tls }}
```

Comment:
Not sure whether it's needed to be able to auth to Dex on HTTPS. @iuriaranda since you created it, can you elaborate?